xbmc/pvr/providers/PVRProviders.h

```diff
@@ -0,0 +1,125 @@
+/*
+ *  Copyright (C) 2012-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "XBDateTime.h"
+#include "pvr/providers/PVRProvider.h"
```

Comment:
do you really need to include this header, as you are forward declaring CPVRProvider later in this file? 

ProviderUpdateMode can be forward declared as it is an enum class.

Reply:
Done